ci-operator/step-registry/openshift/e2e/gcp/csi/openshift-e2e-gcp-csi-workflow.yaml

```diff
@@ -1,14 +1,15 @@
 workflow:
   as: openshift-e2e-gcp-csi
   steps:
+    allow_best_effort_post_steps: true
```

Comment:
does this have any side effect on other steps/chains being executed in post step?